src/main/java/interface_adapters/AppManager.java

```diff
@@ -52,7 +58,7 @@ public void run(Map<String, Window> windows){
     /**
      * Shows the start screen window.
      */
-    public void showStartScreenWindow(){
+    public void showStartScreenWindow() throws ParseException {
```

Comment:
Not completely sure, but why do all of these methods throw ParseException?

Reply:
The code was giving me an error and said I needed to add it.